docs/GettingStarted.md

```diff
@@ -1 +1,25 @@
-# Getting Started
\ No newline at end of file
+# Getting Started
+
+## Solution contents
```

Comment:
We could maybe have a little introduction paragraph here, something like `Thank you for using the Component Library, this page covers the detail you'll need to get up and running' ?